packages/firebase_mlkit_language/README.md

```diff
@@ -0,0 +1,111 @@
+# ML Kit Natural Language Plugin
+
+![Pub](https://img.shields.io/pub/v/firebase_mlkit_language.svg?color=orange)
+
+A Flutter plugin to use the [ML Kit Natural Language for Firebase API](https://firebase.google.com/docs/ml-kit/).
+
+For Flutter plugins for other Firebase products, see [FlutterFire.md](https://github.com/flutter/plugins/blob/master/FlutterFire.md).
+
+*Note*: This plugin is still under development, and some APIs might not be available yet. [Feedback](https://github.com/rishab2113/firebase_mlkit_language/issues) and [Pull Requests](https://github.com/rishab2113/firebase_mlkit_language/pulls) are most welcome!
+
+## Usage
+
+To use this plugin, add `firebase_mlkit_language` as a [dependency in your pubspec.yaml file](https://flutter.io/platform-plugins/). You must also configure Firebase for each platform project: Android and iOS (see the example folder or https://codelabs.developers.google.com/codelabs/flutter-firebase/#4 for step by step details).
+
+### iOS
+Versions `1.0.0+` use the latest ML Kit for Firebase version which requires a minimum deployment
```

Comment:
I'm not sure if this is copy/pasted but I don't think you meant to reference this version